tests/compile_upp.sh

```diff
@@ -123,3 +145,8 @@ make install
 
 rm -rf $PATHTR/exec && mkdir -p $PATHTR/exec
 cp $prefix/bin/upp.x $PATHTR/exec/.
+
+cp $prefix/bin/upp.x $PATHTR/exec/.
```

Comment:
@SamuelTrahanNOAA It looks this line is duplicate of line 147.

Reply:
I removed it.